recipes/pyhecdss/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyhecdss" %}
+{% set version = "0.2.4" %}
+
+package:
+  name:           "{{ name|lower }}"
+  version:        "{{ version }}"
+
+source:
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/CADWRDeltaModeling/pyhecdss/tarball/{{version}}
+
+build:
+  number: 0
+  #skip:           true
+  skip: true # [osx]
+  skip: true # [py<35]
+  skip: true # [win and vc<14]
+  script: LDSHARED=g++ python setup.py install --single-version-externally-managed --record=record.txt # [unix]
+  script: python setup.py install --single-version-externally-managed --record=record.txt # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python
+    - setuptools
+    - numpy {{ numpy }}
+    - swig
+    - pandas >=0.23
+  run:
+    - python >=3.7
+    - numpy  >=1.11
+    - pandas >=0.23
+
+test:
+  imports:
+    - pyhecdss
+  source_files:
+    - tests
+  requires:
+    - pytest
+    - pytest-cov
+  commands:
+    - (setlocal && cd tests && pytest && endlocal) # [win]
+    - (cd tests; pytest) # [unix]
+about:
+  home:           "https://github.com/CADWRDeltaModeling/pyhecdss.git"
+  license:        "MIT"
+  license_family: "MIT"
+  summary:        "Read HECDSS timeseries as pandas DataFrame"
```

Comment:
There needs to be an `extras` section with the maintainers. Please see the example